fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/ComponentTooltipAppenderRegistry.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.item.v1;
+
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.component.ComponentType;
+import net.minecraft.item.tooltip.TooltipAppender;
+
+import net.fabricmc.fabric.impl.item.ComponentTooltipAppenderRegistryImpl;
+
+/**
+ * A registry of {@link TooltipAppender} item components. Adding your item component to this registry will render the
+ * item component to the tooltip of them item when it is present, in a location relative to other item components.
+ */
+@ApiStatus.NonExtendable
+public interface ComponentTooltipAppenderRegistry {
+	static ComponentTooltipAppenderRegistry getInstance() {
+		return ComponentTooltipAppenderRegistryImpl.INSTANCE;
+	}
```

Comment:
I think you can just use static methods here. New APIs don't really use this pattern